tasks/main-blivet.yml

```diff
@@ -82,6 +82,22 @@
 - debug:
     var: _storage_volumes
 
+- name: get required packages
+  blivet:
+    pools: "{{ _storage_pools }}"
+    volumes: "{{ _storage_volumes }}"
+    use_partitions: "{{ storage_use_partitions }}"
+    disklabel_type: "{{ storage_disklabel_type }}"
+  register: package_info
+
+- name: make sure required packages are installed
+  package:
+    name: "{{ pkg_name }}"
+    state: present
+  loop: "{{ package_info.packages }}"
+  loop_control:
+    loop_var: pkg_name
```

Comment:
the package module can accept a list, so you don't need to iterate. See the 3rd item in https://github.com/oasis-roles/meta_standards#ansible-best-practices

Reply:
Ah, good. I did not like having to do them one at a time, but the module docs for `package` don't mention passing a list.

Reply:
True, it is undocumented. But I don't think it will go away. We have more places using it already.

Reply:
Fixed in latest commit (f94da5a).